polaris.dme

```diff
@@ -5,6 +5,144 @@
 // END_INTERNALS
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"

```

Comment:
`FILE_DIR`s are bad.
How to prevent DM from adding these:
```
1. Open up DreamMaker
2. Build > Preferences for polaris.dme...
3. Uncheck "Automatically set FILE_DIR for sub-directories"
```

Reply:
Thanks luv